actions/changelog-generate/action.yaml

```diff
@@ -50,24 +58,28 @@ runs:
         fi
       shell: bash
     - name: Generate latest only changelog
-      uses: orhun/git-cliff-action@v4
+      uses: orhun/git-cliff-action@v4.5
       id: git-cliff-latest
       with:
         config: tmp_cliff.toml
-        args: "--unreleased --tag ${{ inputs.tag }} --verbose"
+        args: "--unreleased --tag ${{ inputs.tag }} ${{ inputs.verbose == 'true' && '--verbose' || '' }} ${{ inputs.additional-cliff-args }}"
       env:
+        # I am not a fan of this, cuz this does not reflect what this step is supposed to do
+        # i.e. just dumping the latest changelog into GHA context
+        # BUT, it works and the file will be written over in the next step anyway
+        OUTPUT: ${{ inputs.changelog-file }}
```

Comment:
Is this still necessary? It seems like there is a default file that git-cliff writes to: https://github.com/orhun/git-cliff-action/blob/98c93442bb05a455a77bee982867857ae748eeea/run.sh#L21

Reply:
that file would have to be explicitly removed after creation, otherwise it would end up in the diff.
I dont really mind adding sth to the rm statement in the cleanup step